hc-256/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "hc-256"
-version = "0.0.0"
+version = "0.0.1"
 authors = ["Eric McCorkle <eric@metricspace.net>"]
-license = "BSD"
+license = "MIT OR Apache-2.0"
```

Comment:
Permission to relicense was granted here: https://github.com/RustCrypto/stream-ciphers/pull/8#issuecomment-453798067